README.md

```diff
@@ -337,11 +337,27 @@ Facebook::Messenger::Profile.set({
 See Facebook's documentation on [Messaging Policy Enforcement](https://developers.facebook.com/docs/messenger-platform/reference/webhook-events/messaging_policy_enforcement)
 
 ```ruby
-Bot.on :'policy_enforcement' do |referral|
+Bot.on :policy_enforcement do |referral|
   referral.action # => 'block'
   referral.reason # => "The bot violated our Platform Policies (https://developers.facebook.com/policy/#messengerplatform). Common violations include sending out excessive spammy messages or being non-functional."
 end
 ```
+
+#### Handle the Handover Protocol
+
+See Facebook's documentation on [Handover Protocol](https://developers.facebook.com/docs/messenger-platform/handover-protocol)
+
+```ruby
+Bot.on :standby do |standby|
+  standby.id                     # => { 'id' => '2015573629214912' }
+  standby.time                   # => 2016-04-22 21:30:36 +0200
+  standby.events.each do |event| # => array of events received in the standby channel which could be
```

Comment:
I think more code showing what each `event` looks like would be helpful. e.g. what are the attributes for `message_reads` or `messages`, etc. So if someone has to implement this on their server, they could get a good idea of what properties and attributes are available and required for them to use.